lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/cc/QtiImport.java

```diff
@@ -181,7 +181,21 @@ public String guessQuestionType (Node itemnode) {
 		return "Fill In the Blank";
 
 	}
-	
+
+	// response_fin is for numeric response
+	// matching has an explicit type, so just need to see if true false
+	NodeList fin=((Element)itemnode).getElementsByTagName("render_fin");
+	if (fin != null && fin.item(0) != null) {
+
+	    NodeList varequal=((Element)itemnode).getElementsByTagName("varequal");
+	    if (varequal != null && varequal.item(0) != null)
+		return "Numeric Response";
+	    NodeList varsubstring=((Element)itemnode).getElementsByTagName("varsubstring");
+	    if (varsubstring != null && varsubstring.item(0) != null)
```

Comment:
Braces around the ifs make for less mistakes down the line.

Reply:
Solved!